catalog/utilities/media_props_gen/column_parser.py

```diff
@@ -0,0 +1,125 @@
+import ast
+import copy
+from pathlib import Path
+
+
+COLUMNS_PATH = Path(__file__).parents[2] / "dags" / "common" / "storage" / "columns.py"
+
+COLUMNS_URL = "https://github.com/WordPress/openverse/blob/main/catalog/dags/common/storage/columns.py"  # noqa: E501
+
+COLUMN = {
+    "python_type": None,
+    "name": None,
+    "db_name": None,
+    "nullable": None,
+    "required": False,
+    "upsert_strategy": "newest_non_null",
+    "base_column": None,
+}
+
+COLUMN_PROPS = COLUMN.keys()
+
+
+def file_to_ast(file_name) -> ast.Module:
+    with open(file_name) as f:
+        file_contents = f.read()
+    return ast.parse(file_contents)
+
+
+CODE = file_to_ast(COLUMNS_PATH)
```

Comment:
Since `COLUMNS_PATH` is a `pathlib.Path` object, this can just be:
```suggestion
CODE = ast.parse(COLUMNS_PATH.read_text())
```